docroot/themes/humsci/humsci_basic/src/scss/components/_buttons.scss

```diff
@@ -28,6 +28,12 @@ $buttons: (
     .hb-dark-inversion .views-element-container & {
       @include hb-global-color('color', 'white');
     }
+
+    @include hb-traditional {
+      .ht-pairing-warbler & {
+        color: white !important;
```

Comment:
I'm guessing  you had no other option but to use !important here... this site is so hard for overrides.

Reply:
Yes, overrides in this case are hard, the clases used for setting the colors are really specific 🤷🏽‍♂️